docs/website-navigation.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Web Navigation, Structure, and Menus
+description: There are several UI elements that you can configure to help users navigate your MyST website.
+thumbnail: thumbnails/table-of-contents.png
+---
+
+In the [default MyST templates](./website-templates.md) there are several sections of the page that help your users navigate within and between pages.
+Here's how you can configure them.
+
+## Website Layout
+
+The default MyST themes are divided into five main sections:
+
+:::{figure} images/website-layout.svg
+:width: 400px
+The layout of the default MyST web themes.
+Other themes may have slightly different structure.
+:::
+
+- **[Header](#navigation:header)**: Contains top-level site navigation.
+- **[Primary sidebar](#navigation:sidebar-primary)**: Cross-page navigation defined in the [Table of Contents](./table-of-contents.md).
+- **[Content window](#navigation:content-window)**: Contains content and metadata for the current page.
+- **[Secondary sidebar](#navigation:sidebar-secondary)**: Contains in-page navigation and links.
+- **[Footer](#navigation:footer)**: (work in progress) Contains more in-depth site navigation.
+
+(navigation:header)=
+## Header
+
+
+(site-navigation)=
+
+### Site navigation
+
+In addition to [your MyST document's Table of Contents](./table-of-contents.md), you may specify a top-level navigation for your MyST site.
+These links are displayed across all pages of your site, and are useful for quickly jumping to sections of your site, or for external links.
+
+Specify top-level navigation with the `site.nav` option, and provide a collection of navigation links similar to [how the Table of Contents is structured](./table-of-contents.md). For example:
+
+```{code-block} yaml
```

Comment:
@choldgraf i was able to find this section but what i had a hard time with is understanding how and when to use url vs file. 

in the example below you have url associated with https links to mystmd.org 

i suggest expanding this example to also include some urls associated with files in the mystmd docs such as a ipynb file and a markdown file.

then please add just a few sentences that clarify what @fwkoch  clarified for me [here](https://github.com/pyOpenSci/peer-review-metrics/pull/27/files#r1712550491) 

which essentially is 

1. if you use file: document.ipynb then it will allow you to provide a downloadable file
2. if you use url: document it will create a link to document.xxx 

it's confusing to me to think about 
1. i have a jupyter notebook that will eventually be a html file. what is the path to that file?
2. intuitivaly i want to use file: because that suggests to me that i'm telling mystmd to find the `document.ipynb` file that will have some html representation in the build
3. because the example shows .org urls i just assumed that url was for an external url not an in project url. 

i hope that helps!! it's working now except i want to get rid of the sidebar toctree now that i have a top bar one working!

Reply:
Thanks for this feedback! I gave it another shot at the commit below.
Do you think this commit clarifies it?

https://github.com/jupyter-book/mystmd/pull/1446/commits/b7dd89a14bc33f33271f67337376b0818e6370cc